docs/src/stream.rst

```diff
@@ -206,8 +206,8 @@ API
         Relying too much on this API is not recommended. It is likely to change
         significantly in the future.
 
-        Currently this only works on Windows and only for
-        :c:type:`uv_pipe_t` handles.
+        Currently only works on Windows for :c:type:`uv_pipe_t` handles.
+        On UNIX platforms, all :c:type:`uv_stream_t` handles are supported.
```

Comment:
Yes, the upcoming one. TBH I'm not sure what this qualifies for, I'll leave it up to you :-)